addon/-private/ember-internals.js

```diff
@@ -0,0 +1,20 @@
+const __EMBER_METAL__ = Ember.__loader.require('@ember/-internals/metal');
```

Comment:
This works in Ember 3.4+

Reply:
From looking here it seems that its from Ember 3.10+
https://github.com/emberjs/ember.js/blob/v3.10.0/packages/@ember/-internals/metal/index.ts